byzcoin/contracts.go

```diff
@@ -24,6 +24,8 @@ import (
 type Contract interface {
 	// Verify returns nil if the instruction is valid with regard to the signature.
 	VerifyInstruction(ReadOnlyStateTrie, Instruction, []byte) error
+	// Verify returns nil if the instruction is valid with regard to the signature.
+	VerifyDeferedInstruction(ReadOnlyStateTrie, Instruction, []byte) error
```

Comment:
`VerifyDeferredInstruction` is called on each instruction of the proposed transaction, which should be implemented by each contract that wants to support a deferred instruction. We can't make it global and implement it inside the DeferredContract since some contract may have implemented their custom `VerifyInstruction`, thus needing to also implement their custom version of `VerifyDeferredInstruction`.